wdqs-frontend/latest/Dockerfile

```diff
@@ -34,6 +34,7 @@ FROM nginx:stable-alpine as final
 WORKDIR /usr/share/nginx/html
 
 COPY --from=builder /tmp/wikidata-query-gui-master /usr/share/nginx/html
+RUN echo "" > style.css
```

Comment:
Are you sure this works? I seem to remember stale `style.css` files interfering with the automatic `style.less` rebuild.

Reply:
On testing it works fine for me.
```
less.js:860 Rendered http://localhost:8282/style.less successfully.
less.js:860 CSS for http://localhost:8282/style.less generated in 153ms
less.js:860 Less has finished. CSS generated in 153ms
```

Reply:
The less building also seemed to work fine for me, but now i don't get the 404 from the style.css.

An alternative could be to remove the loading of style.css, but this seems to work fine for now.